server/src/main/java/org/elasticsearch/cluster/routing/ShardRouting.java

```diff
@@ -748,7 +745,7 @@ public String shortSummary() {
         if (this.unassignedInfo != null) {
             sb.append(", ").append(unassignedInfo.toString());
         }
-        if (expectedShardSize != UNAVAILABLE_EXPECTED_SHARD_SIZE) {
+        if (expectedShardSize != UNAVAILABLE_EXPECTED_SHARD_SIZE && state != ShardRoutingState.STARTED) {
```

Comment:
I find this counter intuitive. If there is a shard size, I think we'd want the short summary and toString to contain it?

Reply:
I believe there was an integration state checking the cluster state that relied on the `toString` output and it was failing because of the changed format. I believe it's should be ok to remove the check after we made sure `expectedShardSize` is set only for searchable snapshots.

Reply:
Fixed it!